doc/build/dts/intro-syntax-structure.rst

```diff
@@ -51,7 +51,9 @@ The above tree has three nodes:
 Nodes can be assigned *node labels*, which are unique shorthands that refer to
 the labeled node. Above, ``a-sub-node`` has the node label
 ``subnode_nodelabel``. A node can have zero, one, or multiple node labels. You
-can use node labels to refer to the node elsewhere in the devicetree.
+can use node labels to refer to the node elsewhere in the devicetree. Note that
```

Comment:
Because *node label* is a Zephyr defined construct, what about changing the name to match the API?